nucypher/characters/control/interfaces.py

```diff
@@ -125,6 +125,37 @@ def revoke(self, label: bytes, bob_verifying_key: bytes):
         response_data = {'failed_revocations': len(failed_revocations)}
         return response_data
 
+    def decrypt(self,
+                label: bytes,
+                policy_encrypting_key: bytes,
```

Comment:
Alice can derive herself the `policy_encrypting_key` using `self.character.get_policy_pubkey_from_label(label)`, so you can get rid of this arg.